src/components/ApplicationForm/Skills.js

```diff
@@ -58,6 +67,27 @@ export default ({ errors, formInputs, onChange, role, handleResume }) => {
 
       <FormSpacing>
         <QuestionHeading>question 12</QuestionHeading>
+        <SubHeading>
+          How many hackathons have you attended (both online and in-person)?
+          <Required />
+        </SubHeading>
+        {errors?.hackathonsAttended && <ErrorMessage>{errors?.hackathonsAttended}</ErrorMessage>}
```

Comment:
Errors are handled in `validation.js`. 
```suggestion
```
There's already a validator defined for # hackathons in `validation.js`. Please move the validator from `basicInfo` to `skills` in the validation file.
```
hackathonsAttended: noEmptyFunction,
```